packages/cli/src/commands/hydrogen/deploy.ts

```diff
@@ -54,10 +55,7 @@ export const deploymentLogger: Logger = (
 export default class Deploy extends Command {
   static description = 'Builds and deploys a Hydrogen storefront to Oxygen.';
   static flags: any = {
-    'env-branch': Flags.string({
```

Comment:
Removing this would be considered a breaking change. Could we keep this for now but mark it as deprecated and suggest to use `env-handle` instead?

Thinking about this more: I don't think we can remove this at all as it's the only thing you'd be able to use in a CI environment.

Reply:
I meant that a user could have custom logic for `--env-branch` to specify a particular environment outside of automatically picking up the branch. Isn't the most likely scenario but because we previously offered it we'll need to continue to offer it until we want to ship a major version.

Reply:
Can this just be a breaking change then if we're only doing monthly releases? It has to be breaking at some point anyways, will delaying that make this better somehow?